src/test/scala/nl.knaw.dans.easy.multideposit/CustomMatchers.scala

```diff
@@ -52,4 +53,17 @@ trait CustomMatchers {
     }
   }
   def equalTrimmed(right: Iterable[Node]) = new EqualTrimmedMatcher(right)
+
+  class ContainAllTrimmed(right: Iterable[Node]) extends Matcher[Iterable[Node]] {
+    override def apply(left: Iterable[Node]): MatchResult = {
+      MatchResult(
+        {
+          left.size == right.size && left.map(l => right.exists(l ==)).forall(true ==)
+        },
+        s"$left did not contain the same elements as $right",
+        s"$left did contain the same elements as $right"
+      )
+    }
+  }
```

Comment:
Maybe print which elements were only present in left and/or right?

Reply:
thanks for the suggestion. I'm not entirely happy with the solution for now, but this is the easiest solution for now. Maybe I'll find the 'official' ScalaTest way to do this correctly later.